setting-up/migrate.md

```diff
@@ -0,0 +1,125 @@
+export OLD_APPLICATION_DIR=$PWD/appsmith-docker-stack/appsmith
+export NEW_APPLICATION_DIR=$PWD/appsmith-fat-container
+
+# Create configuration folder
+mkdir -p $NEW_APPLICATION_DIR/stacks/configuration
+
+# Copy environment variable from docker.env & encryption.env 
+cat $OLD_APPLICATION_DIR/docker.env $OLD_APPLICATION_DIR/encryption.env >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
+```
+* We also introduce a new application stack in Appsmith system - `Realtime Service (RTS)`. Please run the below command to configure the backend URL for `RTS`
+```
+# Add backend URL configuration
+echo APPSMITH_API_BASE_URL=http://localhost:8080 >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
+```
+
+*Note: In our new image, we have used Redis and MongoDB service inside the container as default services. Therefore, if you do not use external services for Redis and MongoDB, you will need to change 2 environment variables in the `docker.env` file (Replace `<MONGO_USERNAME>` and `<MONGO_PASSWORD>` with existing `MONGO_INITDB_ROOT_USERNAME`  and `MONGO_INITDB_ROOT_PASSWORD`  in the `docker.env` file)*
+```
+APPSMITH_MONGODB_URI=mongodb://<MONGO_USERNAME>:<MONGO_PASSWORD>@localhost/appsmith
+APPSMITH_REDIS_URL=redis://127.0.0.1:6379
+```
+### 3. Export https config & certificate (Optional)
+
+* If you have deployed Appsmith with custom domain, you might want to move your certificate to the new container
+
+```
+# Add custom domain to configuration file
+echo APPSMITH_CUSTOM_DOMAIN=<YOUR-DOMAIN> >> $NEW_APPLICATION_DIR/stacks/configuration/docker.env
+
+# Create Letsencrypt folder in Docker mounting point
+mkdir -p $NEW_APPLICATION_DIR/stacks/letsencrypt
+
+# Copy Letsencrypt certificate to new Letsencrypt folder
+sudo cp -rf $OLD_APPLICATION_DIR/data/certbot/conf/* $NEW_APPLICATION_DIR/stacks/letsencrypt
+```
+
+### 4. Setup new Appsmith with Fat container
+* Before you can start new appsmith docker, please shutdown current appsmith application first
+```
+cd $OLD_APPLICATION_DIR
+docker-compose down
+```
+* Now, you can start to set up Appsmith with new Docker image. Please follow this guideline to start up a new container: [https://github.com/appsmithorg/appsmith/blob/release/deploy/docker/README.md](https://github.com/appsmithorg/appsmith/blob/release/deploy/docker/README.md)
+* Please note that you must create new docker-compose.yml in `appsmith-fat-container` folder
+
+### 5. Import database
+
+* After successfully starting up new container, you can import your previous data by exported file from the above step
+* By using `Appsmith CLI`, new feature we have nearly introduced in the new Docker image, you can import all data with only one command
+
+```
+# Create folder restore
+mkdir -p $NEW_APPLICATION_DIR/stacks/data/restore
+
+# Copy exported file 
+sudo cp $OLD_APPLICATION_DIR/data/mongo/db/backup/appsmith-data.archive $NEW_APPLICATION_DIR/stacks/data/restore/appsmith-data.archive
```

Comment:
I see that we are not including `sudo` in the docker/docker-compose commands, which makes me wonder if we should included here at all? Can we remove it?